src/components/BooksForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const BooksForm = () => (
```

Comment:
- [ ] Since the BookForm component will soon be connected with Redux and therefore it will be aware of Redux's existence thus it's considered a Container component and should be placed at the `containers` directory.